LiskHQ/lisk-desktop
# Fix timestamp calculation and account creation bugs

### What was the problem?

This PR resolves #5266, #5236

### How was it solved?

- Fix last forger timestamp calculation
- Fix account creation to not use legacy address derivation
- Bug fixes related to blockchain application and few other minor UI bugs.

### How was it tested?

- view validators tab and go over some validator profile and check the last forged timestamp
- Use latest protocol with CDP for account creation
- CSS and text changes for UI bugs
